libs/app-shell/src/lib/sitemap.js

```diff
@@ -24,7 +24,7 @@ const sitemap = [
       },
       {
         label: 'JBrowse',
-        route: '/jbrowse/index.html?data=data%2FHomo%20sapiens',
```

Comment:
@scottcain this goes in the sitemap xml file for google to use so you might not want to change this if the direct hit in a browser still works.

Reply:
This is actually reverting a change I made last week when I didn't understand the nature of the problem.

Reply:
oh ok